index.html

```diff
@@ -8,44 +8,275 @@
+                        <a href="#">
+                            <img src="./img/m.png" alt="m link">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./img/github.png" alt="twitter link">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./img/m2.png" alt="m2 link">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section id="works">
+            <div class="works-item works-item-1">
+                <div class="works-item-image-1"></div>
+                <div class="works-item-info-block">
+                    <div class="works-item-project-info">
+                        <h2 class="works-item-title"> Tonic </h2>
+                        <div class="works-item-project-info-frame">
+                            <p class="works-item-project-info-canopy works-item-project-info-items"> 
+                                <span>CANOPY</span> 
+                            </p>
+                            <img src="img/Counter.png" alt="..." class="works-item-project-info-point-1 works-item-project-info-items">
+                            <p class="works-item-project-info-type"> 
+                                <span> Back End Dev</span> 
+                            </p>
+                            <img src="img/Counter.png" alt="..." class="works-item-project-info-point-2 works-item-project-info-items">
+                            <p class="works-item-project-info-year"> 
+                                <span>2022</span> 
+                            </p>
+                        </div>
+                    </div>
+                    <p class="works-item-project-description"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="works-item-project-tecnologies">
+                        <li class="works-item-project-tecnologies-item works-item-project-item-html"> 
+                            <span> html </span> 
+                        </li>
+                        <li class="works-item-project-tecnologies-item works-item-project-item-css"> 
+                            <span> css </span>
+                        </li>
+                        <li class="works-item-project-tecnologies-item works-item-project-item-javascript"> 
+                            <span> javaScript </span>
+                        </li>
+                    </ul>
+                    <div id="works-item-project-action">
+                        <button class="works-item-project-action-button">
+                            <span>  See Project </span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <div class="works-item works-item-2">
```

Comment:
[Optional] As the previous reviewer mentioned it's important to stick with the Figma design as much as possible. So please make sure to put some space between the work cards.

![image](https://user-images.githubusercontent.com/34779928/192084157-76157b18-dbb3-4388-9a97-92fe8deb892b.png)